index.html

```diff
@@ -107,6 +107,99 @@ <h4>Skills</h4>
+          <div class="projects-seection">
+            <a href=""><img src="images/Img-Placeholder.png" alt=""></a>
+            <h2 class="post-tittle">Multi-Post Stories</h2>
+            <div class="support-text">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard dummy text ever since the 1500s,
+                when an unknown printer took a standard dummy text.</p>
+              <ul >
+                <li> css </li>
+                <li> html</li>
+                <li>bootstrap</li>
+                <li>Ruby </li>
+              </ul>
+              <button class="bttn" type="button">See Project</button>
+          </div>
+        </div>
+        <div class="projects-seection">
+          <a href=""><img src="images/Img-Placeholder.png" alt=""></a>
+          <h2 class="post-tittle">Multi-Post Stories</h2>
+          <div class="support-text">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard dummy text ever since the 1500s,
+                when an unknown printer took a standard dummy text.
+              </p>
+              <ul>
+                <li> css </li>
+                <li> html</li>
+                <li>bootstrap</li>
+                <li>Ruby </li>
+              </ul>
+            <button class="bttn" type="button">See Project</button>
+          </div>
+        </div>
+          <div class="projects-seection">
+            <a href=""><img src="images/Img-Placeholder.png" alt=""></a>
+            <h2 class="post-tittle">Multi-Post Stories</h2>
+            <div class="support-text">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.<br>
+                has been the industry's standard dummy text ever since the 1500s,<br>
+                when an unknown printer took a standard dummy text.
+              </p>
+              <ul>
+                <li> css </li>
+                <li> html</li>
+                <li>bootstrap</li>
+                <li>Ruby </li>
+              </ul>
+              <button class= "bttn" type="button">See Project</button>
+          </div>
+        </div>
+      </section>
+      <section class="contact-section">
+      <div class="frame42">
+        <h2>CONTACT US</h2>
+        <p>
+          If you have an application you are <br>
+          interested in developing, a feature that you <br>
+          need built or a project that needs coding. <br>
+          I’d love to help with it
+        </p>
+      </div>
+      <form>
+        <input type="text" name="text" placeholder="Text Input">
+        <input type="email" name="email" id="email" placeholder="test@gmail.com">
+        <textarea name="text" id="text" cols="30" rows="5">Enter text here</textarea>
+        <button type="submit">Send</button>
+      </form>
     </section>
```

Comment:
- [ ] Please remove the contact form section from this milestone is not part of the requirement for this milestone.